pkgs/by-name/ta/tana/package.nix

```diff
@@ -0,0 +1,114 @@
+{ xorg
+, stdenv
+, lib
+, alsa-lib
+, at-spi2-atk
+, atkmm
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, gtk3
+, libdrm
+, libglvnd
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, pango
+, systemd
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, makeWrapper
+, wrapGAppsHook
+}:
+let
+  xorgLibs = with xorg; [
+    libX11
+    libxcb
+    libXcomposite
+    libXdamage
+    libXext
+    libXfixes
+    libXrandr
+  ];
+  libs = [
+    alsa-lib
+    at-spi2-atk
+    atkmm
+    cairo
+    cups
+    dbus
+    expat
+    glib
+    gtk3
+    libdrm
+    libglvnd
+    libxkbcommon
+    mesa
+    nspr
+    nss
+    pango
+  ];
+  buildInputs = xorgLibs ++ libs;
+  version = "1.0.15";
+in
+stdenv.mkDerivation {
+  pname = "tana";
+  inherit version buildInputs;
+
+  src = fetchurl {
+    url = "https://github.com/tanainc/tana-desktop-releases/releases/download/v1.0.15/tana_${version}_amd64.deb";
+    hash = "sha256-94AyAwNFN5FCol97US1Pv8IN1+WMRA3St9kL2w+9FJU=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    makeWrapper
+    stdenv.cc.cc.lib
+    wrapGAppsHook
+  ];
+
+  # Needed for Zygote
+  runtimeDependencies = [
+    systemd
+  ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
```

Comment:
Break those phases something?

Reply:
I'm not sure I understand? Those phases are not needed for this derivation since we're not building from source.. do you want me to explicitly set the phases in the phases variable or do you want me to keep those phases anyway?

Reply:
you can remove both of them and nothing will break. i think the default configure looks for ./configure and build looks for a makefile and if there isn't one, then they don't do anything. so disabling them is not required.